onnxruntime/core/providers/dml/DmlExecutionProvider/src/Operators/OperatorRegistration.h

```diff
@@ -8,8 +8,9 @@ interface IMLOperatorKernel;
 class MLOperatorKernelCreationContext;
 
 // Forward declares an external creation function.
-#define DML_OP_EXTERN_CREATION_FUNCTION(operatorName) extern void CALLBACK Create##operatorName(IMLOperatorKernelCreationContext* kernelInfo, IMLOperatorKernel** opKernel)
+#define DML_OP_EXTERN_CREATION_FUNCTION(operatorName) extern void CALLBACK Create##operatorName(IMLOperatorKernelCreationContext* kernelInfo, IMLOperatorKernel** opKernel);
```

Comment:
[nit] semicolon at the end of this will be an extra semicolon because caller of this macro adds semicolon after the usage.

DML_OP_EXTERN_CREATION_FUNCTION(Resize13);